src/components/display.js

```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
+import ReactTooltip from 'react-tooltip';
```

Comment:
I think you forgot to push your changes to package.json. react-tooltip is a new dependency.